src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -66,6 +70,47 @@ public static MapRouletteClient instance(final MapRouletteConfiguration configur
         return null;
     }
 
+    /**
+     * This method writes challenge ids to a file.
+     *
+     * @param challengeId
+     *            challenge id of the newly created MapRoulette challenge.
+     * @param projectId
+     *            project id of the MapRoulette challenge.
+     */
+    public static void writeChallengeIdsToFile(final long challengeId, final long projectId)
+    {
+        outputPath.ifPresent(fileName ->
```

Comment:
Should we enforce the filename for consistency sake? That way, the user will only need to specify the output path, and we'll be in charge of setting the filename: `challenges.txt`.